kahuna/public/js/components/gu-date-range/gu-date-range.js

```diff
@@ -90,7 +90,7 @@ guDateRange.directive('guDateRange', [function () {
             });
 
             $scope.$watch('pikaStartValue', function (pikaStartValue) {
-                var date = pikaStartValue && new Date(pikaStartValue);
+                var date = (pikaStartValue && new Date(pikaStartValue)) || new Date();
```

Comment:
This is currently passing an empty string about, which is dealt with silently until now.